packages/sdk/src/sdk-blockchains/ethereum/bid.ts

```diff
@@ -138,12 +159,68 @@ export class EthereumBid {
 			multiple: collection.type === "ERC1155",
 			maxAmount: item.supply,
 			baseFee: await this.sdk.order.getBaseOrderFee(),
+			getConvertableValue: this.getConvertableValue,
+			convert: this.convertCurrency,
 			submit,
 		}
 	}
 
+	getConvertMap() {
+		return {
+			[`ETHEREUM:${this.sdk.balances.getWethContractAddress()}`]: "ETH",
```

Comment:
do we have utility like `toUnionAddress` 
it's not scalable when we doing it by hands every time

Reply:
This convert map can be static field in EthereumBid class,
there's no `this` inside and it's immutable